### PR TITLE
Added suppression rules for python

### PR DESCRIPTION
**Pull request by sima-zhu**

### CODE REVIEW

**Review by danieljurek — APPROVED** [review]

**Comment by danieljurek** [comment]
LGTM. Do you have examples of a build where this passes? I didn't see one when I searched the checks associated w/ the PR.

**Comment by sima-zhu** [comment]
> LGTM. Do you have examples of a build where this passes? I didn't see one when I searched the checks associated w/ the PR.

@danieljurek The cred scan is in aggregate-report. I have one triggered here: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=740233&view=results

File: eng/CredScanSuppression.json [hunk 1]
```diff
@@ -1,4 +1,48 @@
...
+                "NewPassword1!",
+                "MyStr0ngP4ssword",
+                "secret_password",
+                "secret",
+                "123",
+                "connectionstring",
+                "123_abc",
+                "pa$$w0rd",
+                "Password1!!!",
+                "NzhL3hKZbJBuJ2484dPTR+xF30kYaWSSCbs2BzLgVVI1woqeST/1IgqaLm6QAOTxtGvxctSNbIR/1hW8yH+bJg==",
```
Comment (weshaggard) [inline eng/CredScanSuppression.json:28]: Is this really a placeholder? Some of these look like keys.

**Review by weshaggard — APPROVED** [review]
One question but looks good otherwise.

File: eng/CredScanSuppression.json (hunk 1 above)
Comment (sima-zhu) [reply eng/CredScanSuppression.json]: Right, those are symmetricKey. Will remove and work with team to use some static ones.

**Pull request merged**